src/main/java/io/getunleash/impactmetrics/ImpactMetricRegistryAndDataSource.java

```diff
@@ -0,0 +1,4 @@
+package io.getunleash.impactmetrics;
+
+public interface ImpactMetricRegistryAndDataSource
```

Comment:
We needed type definition that includes ImpactMetricRegistry and ImpactMetricsDataSource, since in Java we have config that accepts this. This is new to Java, not in Node.